infra/lib/fhir-converter-service.ts

```diff
@@ -24,6 +24,8 @@ export function settings() {
     memoryLimitMiB: prod ? 8192 : 4096,
     taskCountMin: prod ? 2 : 1,
     taskCountMax: prod ? 30 : 10,
+    // How long this service can run for
+    maxExecutionTimeout: Duration.minutes(15),
```

Comment:
nit: we use `Duration.minutes(15)` in multiple places, which is really the lambda max run duration - would be good to put in a const so we can use throughout the infra

Reply:
We only use in 2 places, and those represent different things: one is the FHIR Converter's ALB timeout, the other is the FHIR server's ALB timeout (which come from a diff repo). Also, they might be different, so it also seems to make sense so keep them separated.

OTOH, we could have a constant to represent the maximum time a lambda can run for, and then have those referencing that. 👍